lib/modules/manager/pip-compile/extract.ts

```diff
@@ -95,8 +99,7 @@ export async function extractAllPackageFiles(
         type: 'constraint',
       });
     }
-    // TODO(not7cd): handle locked deps
-    // const lockedDeps = extractRequirementsFile(content);
+    const lockedDeps = extractRequirementsFile(fileContent)!.deps;
```

Comment:
```suggestion
    const lockedDeps = extractRequirementsFile(fileContent).deps;
```
why the assertion? should required or needs a check.

Reply:
If we were able to successfully parse the header, then deps should exist. Although, there may be files with only header.